src/entity/Source.ts

```diff
@@ -22,6 +22,19 @@ export enum SourceType {
   Squad = 'squad',
 }
 
+interface SourceFlags {
+  featured: boolean;
+  totalViews: number;
+  totalPosts: number;
+  totalUpvotes: number;
+}
+
+const defaultFlags: Omit<SourceFlags, 'featured'> = {
+  totalPosts: 0,
+  totalViews: 0,
+  totalUpvotes: 0,
+};
```

Comment:
Flags should be empty by default, so any defaulting to zero should be handled in other API logic or on graphorm side. 

For example having defaultValues and then updating them after some time would require migration to update all previous rows that had different default value. Which is complicated to do.

Reply:
I actually did that at first but hesitated so I went to the RFC and checked if we specified a default value, found they mentioned 0 so I went ahead and applied it. @DragosIorgulescu thoughts?

Reply:
It can be solved on graphORM side not db side basically.

Reply:
Yeah, I understand, it is just on the RFC, the table was dedicated to the entity, not the gql schema. In any case, I will revert to empty curly brackets similar to our other implementations of flags.